pkg/config/config.go

```diff
@@ -35,7 +34,7 @@ const (
 	defaultNumberOfWorkers      = 1
 	defaultNumberOfQueryWorkers = 8
 	defaultBatchSize            = 64
-	defaultTimeout              = 24 * time.Hour
+	defaultTimeoutInSeconds     = 24 * 60 * 60 // 24 hours
```

Comment:
consider using `time.Hour / time.Seconds` which will give you one hour in seconds, might look better than literal numbers